website/frontend/src/components/blocks/legacy/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { VFC } from 'react';
+import { PAGE_TYPES } from 'consts';
+import { fileNameHandler } from 'helpers';
+import ArrowBack from 'assets/images/arrow_back.svg';
+import ProgramIllustration from 'assets/images/program_icon.svg';
+import close from 'assets/images/close.svg';
+import { Wallet } from '../../Wallet/Wallet';
+import './PageHeader.scss';
+
+type Props = {
+  programName: string;
+  pageType: string;
+  handleClose: () => void;
+};
+
+export const PageHeader: VFC<Props> = ({ programName, pageType, handleClose }) => {
+  let headInscription = 'New request to';
+  let formId = 'message-form';
+  if (pageType === PAGE_TYPES.EDITOR_PAGE) {
+    headInscription = 'Edit code';
+  } else if (pageType === PAGE_TYPES.META_FORM_PAGE) {
+    headInscription = 'Upload metadata';
+    formId = 'meta-form';
+  }
+
+  return (
+    <div className="message-header">
+      <div className="message-header--info">
+        <button
+          type="button"
+          aria-label="arrowBack"
+          onClick={() => handleClose()}
```

Comment:
no need to wrap it with arrow func

Reply:
it's legacy component, I only moved it. It's will remove later.